packages/forms/stories/story-types.ts

```diff
@@ -22,13 +23,16 @@ export interface ICheckboxStoryProps {
 }
 
 export interface IRangeStoryProps {
+  isRegular: boolean;
+  isHidden: false;
```

Comment:
Should the type for this property be `boolean` since it could be `true` and `false`?